vm/qemu/qemu.go

```diff
@@ -146,7 +146,7 @@ var archConfigs = map[string]*archConfig{
 	},
 	"linux/arm64": {
 		Qemu:     "qemu-system-aarch64",
-		QemuArgs: "-machine virt,virtualization=on -machine gic-version=max -cpu cortex-a57 -accel tcg,thread=multi",
+		QemuArgs: "-machine virt,virtualization=on -machine gic-version=max -cpu max,sve=off,pauth=off -accel tcg,thread=multi",
```

Comment:
This is what we already do on https://syzkaller.appspot.com/upstream/manager/ci-qemu-native-arm64-kvm

Reply:
Please add a comment for future us as to why we disable these features.